Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG NODE_VERSION="12.2.0"
+FROM library/node:${NODE_VERSION} AS development
```

Comment:
I tried to google this, but can you explain how `FROM library/node:12.2.0` differs from `FROM node:12.2.0`. Is there any resource/blog you can point me to? 😊

Reply:
Oh, there isn't really a difference, when you omit the image prefix, it defaults to "library" (which refers to images hosted on hub.docker.com). I auto-pilot add the prefix because I also have Docker image repos at home, at work, and on the cloud 😅

Reply:
Thanks for the clarification! Shall we remove the "library" prefix since it's not very common these days? 😊